source/declarations.tex

```diff
@@ -879,33 +879,26 @@
 
 \begin{example}
 \begin{codeblock}
-constexpr int square(int x)
-  { return x * x; }             // OK
-constexpr long long_max()
-  { return 2147483647; }        // OK
-constexpr int abs(int x) {
-  if (x < 0)
-    x = -x;
-  return x;                     // OK
+constexpr int square(int x) {           // OK
+  return x * x;
 }
-constexpr int constant_non_42(int n) {  // OK
-  if (n == 42) {
-    static int value = n;
-    return value;
-  }
-  return n;
+constexpr std::generator<int> iota(int n = 0) {
+    while (true) co_yield n++;          // error: \tcode{co_yield} cannot be used in a constexpr function
 }
-constexpr int uninit() {
-  struct { int a; } s;
-  return s.a;                   // error: uninitialized read of \tcode{s.a}
+constexpr int stinit(int n) {           // \tcode{stinit} is constexpr-usable, but \tcode{stinit}
```

Comment:
The paragraph the example is attached to defines _constexpr-suitable_, so I think that's what you meant.
```suggestion
constexpr int stinit(int n) {           // \tcode{stinit} is constexpr-suitable, but \tcode{stinit}
```